src/js/controller/instream-adapter.js

```diff
@@ -403,6 +403,16 @@ const InstreamAdapter = function(_controller, _model, _view, _mediaPool) {
      * @return {void}
      */
     this.skipAd = function(event) {
+        const autoPauseAds = _model.get('autoPause').pauseAds;
+        const didNotAutostart = _model.get('playReason') !== 'autostart';
```

Comment:
I did initially but felt like keeping the variables positive in the conditional was better but I'm open to change if you feel like it is necessary.

Reply:
I am fine either way

Reply:
I think it's best practice to avoid negative variable names - let's imagine in the future we'd have to add a `else if (!didNotAutostart)`; the double negative gets confusing.

Reply:
Good point. I'll change it.